tracee-ebpf/tracee/tracee.bpf.c

```diff
@@ -118,6 +118,11 @@
 #define SYS_EXIT_GROUP        231
 #define SYS_OPENAT            257
 #define SYS_EXECVEAT          322
+#define CONNECT_SYSCALL       42
```

Comment:
i chose this naming convention because SYS_XXX for these events are already defined in: uapi/linux/net.h

https://elixir.bootlin.com/linux/latest/source/include/uapi/linux/net.h#L28

Reply:
OK, got it.
I don't see that we directly use this header in tracee.bpf.c, so I guess, it is being use by one of the other headers, right?

So maybe to have the same prefix for these macros we should change to "SYSCALL_CONNECT", etc, WDYT?

Reply:
agreed, no problem